public/app/frontend/src/components/file-download/file-download.stories.js

```diff
@@ -0,0 +1,33 @@
+import fileDownload from './file-download.html.twig';
+import './index.js';
+
+export default {
+    title: 'Components/File download',
+    parameters: {
+        layout: 'centered',
+    },
+    argTypes: {
+        format: {
+            options: ['PDF', 'ZIP'],
+            control: { type: 'radio' },
+        },
+    },
+};
+
+const Template = (args) => {
+    return fileDownload(args);
+};
+
+export const Default = Template.bind({});
+Default.args = {
+    format: 'PDF',
+    link: '#',
+    filesize: '167 KB',
+    filename: 'CPR email address list',
```

Comment:
If these are default, should we be specifying initial values that suggest they are not currently set:
```
    filesize: '0 KB',
    filename: '',
```

Reply:
Hi @wilson1000, they're not default values, anything in the .stories.js files are example values that are used when displaying the story. They won't appear outside of storybook.